index.test.js

```diff
@@ -128,3 +128,54 @@ ruleTester.run("component-prefix", plugin.rules["component-prefix"], {
     }
   ]
 });
+
+const isNotAComponent = `
+export class ManifoldSelect {
+  @Prop() answer: number;
+
+  render() {
+    return null;
+  }
+}`;
+
+const componentMissingLogger = `
+import { h, Component, Prop } from '@stencil/core';
```

Comment:
So this doesn’t get evaluated by JS, because `@stencil/core` isn’t a dependency? The ESLint parser supports decorators and stuff out-of-the-box?

Reply:
Yeah, there's no evaluation here, only parsing the text into nodes. And yes, decorators are supported out of the box!